nipype/interfaces/base.py

```diff
@@ -1366,10 +1366,10 @@ def _get_ram_mb(pid, pyfunc=False):
         mem_mb = parent_mem/_MB
 
         # Iterate through child processes
-        for child in parent.children(recursive=False):
+        for child in parent.children(recursive=True):
```

Comment:
why don't you switch runtime_profiler off, because these changes are derived from that, right?